### ISSUE TITLE
Option '-c' not working

### ISSUE THREAD

**Issue by Yrr0r** [shadowsocks/shadowsocks-libev#2142]
### What version of shadowsocks-libev are you using?
shadowsocks-libev 3.2.0
### What operating system are you using?
Tested on Ubuntu 16.04 and MacOS 10.13.6
### What did you do?
Use command
`ss-server -c /Users/steve/Desktop/config.json` or
`ss-server -c ./Desktop/config.json`
On both case when the config file exists or not.

### What did you expect to see?
The program should start normally and printing logs.

### What did you see instead?
The program to display its help message and exit without error (equivalent to `ss-server -h`) when the config file exist, no matter what I put in that JSON, or no matter if that JSON had worked on python shadowsocks before.
And if the config file does not exist in that path, the program will throw an error as following and exit immidiately:
`2018-08-09 00:24:34 ERROR: Invalid config path.` 

I suppose ss-server had attempted loading or opening the config file but somehow exited. I can now only use command line arguments instead of use a config file from my another server. But it successfully loaded the config file in default path in `/etc` when I start it as a service from systemctl.

_Please close this issue if it's still opening and the problem is solved._

**Comment by madeye** [comment]
Post your config file here.

**Comment by Yrr0r** [comment]
I edited the issue to make it following the template.

And I found the problem. It was the config file. I thought it was the reason of the ss-server because it has no error output when a JSON with insufficient arguments is loaded.

 The following JSON worked well:
`{
"server_port":12000,
"password":"******",
"timeout":300,
"method":"chacha20"
}`
But this one produced the error:
`{
"port":12000,
"password":"******",
"timeout":300,
"method":"chacha20"
}`
